graphql2/graphqlapp/contactmethod.go

```
@@ -32,6 +32,39 @@ func (a *ContactMethod) FormattedValue(ctx context.Context, obj *contactmethod.C
 	return formatted, nil
 }
 
+func (a *ContactMethod) LastTestMessageState(ctx context.Context, obj *contactmethod.ContactMethod) (*graphql2.NotificationState, error) {
+	t := obj.LastTestVerifyAt()
+	if t.IsZero() {
+		return nil, nil
+	}
+
+	status, _, err := a.NotificationStore.LastMessageStatus(ctx, "test_notification", obj.ID, t)
+	if status == nil {
+		return nil, nil
+	}
```

This will always ignore the error, put this after the `err != nil` check